src/chain/logging.js

```diff
@@ -1,19 +1,19 @@
 var path = require("path")
 var fs = require("fs")
-var padStart = require("lodash.padstart")
+var _ = require("lodash")
```

Comment:
Why switch from the piece-wise version to the full one? It'd be better to add piece-wise modules than to pack the whole of lodash - it's pretty big, IIRC.

Reply:
ignore this and just focus on the network interface enumeration stuff, then we're :+1: